production/helm/loki/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
-appVersion: 2.8.1
+appVersion: 2.8.2
 version: 5.3.1
```

```suggestion
appVersion: 2.8.1
version: 5.3.2
```
Sorry. It's the other way around :slightly_smiling_face: